carrie_gz/launch/include/spawn_robot.launch.py

```diff
@@ -0,0 +1,143 @@
+"""Spawn a Carrie robot in new Gazebo, also launch the robot_state_publisher."""
+
+import os
+
+from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
+from launch_ros.actions import Node
+from xacro import process_file
+
+PKG_CARRIE_DESCRIPTION = get_package_share_directory('carrie_description')
+PKG_CARRIE_GZ = get_package_share_directory('carrie_gz')
```

Comment:
Should `carrie_description` and `carrie_gz` be launch arguments? I don't expect them to change but it seems a bit strange to just have these essentially be globals.

Reply:
Move these variables into the `get_robot_description` function